### PR TITLE
libct/int: misc cleanups and fixes

### PR DESCRIPTION
**Pull request by kolyshkin**
Please see individual commits for details. Nothing serious, but improved container names and cgroups randomization might help to reduce flakes.

### CODE REVIEW

**Review by AkihiroSuda — APPROVED** [review]

File: libcontainer/integration/template_test.go [hunk 1]
```diff
@@ -213,9 +214,9 @@ func newTemplateConfig(p *tParam) *configs.Config {
 	}
 
 	if p.systemd {
-		id := strconv.FormatUint(rand.Uint64(), 36)
-		config.Cgroups.Name = "test" + id
-		// do not change Parent (see newContainerWithName)
+		id := strconv.FormatInt(-int64(time.Now().Nanosecond()), 36)
```
Comment (fuweid) [inline libcontainer/integration/template_test.go:217]: minus~ good one

Comment (fuweid) [reply libcontainer/integration/template_test.go:217]: Why not just use `rand.Uint64` to prevent from using the same id

Comment (kolyshkin) [reply libcontainer/integration/template_test.go:217]: Because it needs to be seeded somewhere

**Review by mrunalp — APPROVED** [review]

**Comment by kolyshkin** [comment]
this is CI only stuff, so merging

**Pull request merged**